torch/_dynamo/types.py

```diff
@@ -94,3 +94,10 @@ def __call__(
 class ProfilerEndHook(Protocol):
     def __call__(self, record: Any) -> None:
         ...
+
+
+class BytecodeHook(Protocol):
```

Comment:
unused?

Reply:
It is used to specify the protocol. Since no hooks are installed by default, we have to show users the protocol of hooks.